chroma-manager/tests/integration/installation_and_upgrade/test_update_with_yum.py

```diff
@@ -55,6 +55,18 @@ def test_no_retired_repos(self):
         for repo in retired_repos:
             self.assertFalse(repo in current_repos, "Unexpectedly found repo '%s' in %s" % (repo, current_repos))
 
+    def test_obsolete_chroma_diagnostics(self):
+        """Test that chroma-diagnostics has been obsoleted"""
+        servers = config['lustre_servers']
+        addresses = [server['address'] for server in servers]
+        
+        for address in addresses:
+            chroma_diagnostics_result = self.remote_command(address, 'chroma-diagnostics')
+            self.assertEqual(chroma_diagnostics_result.stdout.split('\n')[0], "chroma-diagnostics no longer exists. Please use 'iml-diagnostics' instead.")
+            
+            yum_installed_result = self.remote_command(address, 'yum list installed')
```

Comment:
This is still a lot more work than just using `rpm -q chroma-diagnostics` and testing `yum_installed_result.rc != 0`.

You are essentially dumping a whole database and searching it yourself rather than using the database tool to search it for you. This would be like doing a `mysqldump | grep` rather than using the `mysql` client tool to issue a SQL query.

Reply:
Yeah that makes more since. I thought I had to use yum to check for installed packages, didn't realize I could use rpm like this. Thanks!